hono/src/services/items.ts

```diff
@@ -40,20 +82,23 @@ app.post(
   injectDB,
   async (c) => {
     const body = c.req.valid('json')
+    const res = (
+      await c
+        .get('db')
+        .insert(items)
+        .values({
+          ...body,
+          authorUsername: 'rak3rman', // TODO: remove and replace with author integration
+        })
+        .onConflictDoNothing()
+        .returning()
+    )[0]
 
-    return c.json(
-      (
-        await c
-          .get('db')
-          .insert(items)
-          .values({
-            ...body,
-            authorUsername: 'rak3rman', // TODO: remove and replace with author integration
-          })
-          .onConflictDoNothing()
-          .returning()
-      )[0]
-    )
+    await c.get('db').execute(sql`
+      ANALYZE items;
+    `)
```

Comment:
Discussion pulled from Radical Discord on changes requested.

Running `ANALYZE items;` needs to be non-blocking. Figure out a way to do this for POST and DELETE...

> Well here are the 3 options that come to mind in order of difficulty (ideal to hardest):
> A. find a way to make the ANALYZE call non-blocking and still execute on the db
> B. setup autovacuum and define params such that vacuum/analyze is run automatically
> C. basically recreate autovacuum and have a separate daemon / worker process that can wait for ANALYZE to finish, the hono backend will trigger this service

Reply:
Ended up using a "last_page" attribute instead such that pagination can be traversed linked list style.